.drone.star

```diff
@@ -1156,22 +1156,22 @@ def ocisServer(storage):
       'detach': True,
       'environment' : {
         #'OCIS_LOG_LEVEL': 'debug',
-        'REVA_STORAGE_HOME_DRIVER': '%s' % (storage),
-        'REVA_STORAGE_HOME_DATA_DRIVER': '%s' % (storage),
-        'REVA_STORAGE_OC_DRIVER': '%s' % (storage),
-        'REVA_STORAGE_OC_DATA_DRIVER': '%s' % (storage),
-        'REVA_STORAGE_HOME_DATA_TEMP_FOLDER': '/srv/app/tmp/',
-        'REVA_STORAGE_OCIS_ROOT': '/srv/app/tmp/ocis/root',
-        'REVA_STORAGE_LOCAL_ROOT': '/srv/app/tmp/reva/root',
-        'REVA_STORAGE_OWNCLOUD_DATADIR': '/srv/app/tmp/reva/data',
-        'REVA_STORAGE_OC_DATA_TEMP_FOLDER': '/srv/app/tmp/',
-        'REVA_STORAGE_OWNCLOUD_REDIS_ADDR': 'redis:6379',
-        'REVA_LDAP_IDP': 'https://ocis-server:9200',
-        'REVA_OIDC_ISSUER': 'https://ocis-server:9200',
+        'STORAGE_STORAGE_HOME_DRIVER': '%s' % (storage),
+        'STORAGE_STORAGE_HOME_DATA_DRIVER': '%s' % (storage),
+        'STORAGE_STORAGE_OC_DRIVER': '%s' % (storage),
+        'STORAGE_STORAGE_OC_DATA_DRIVER': '%s' % (storage),
+        'STORAGE_STORAGE_HOME_DATA_TEMP_FOLDER': '/srv/app/tmp/',
+        'STORAGE_STORAGE_OCIS_ROOT': '/srv/app/tmp/ocis/storage/users/nodes/root/',
+        'STORAGE_STORAGE_LOCAL_ROOT': '/srv/app/tmp/ocis/local/root',
```

Comment:
`/srv/app/tmp/ocis/local`

Reply:
was 
`'REVA_STORAGE_LOCAL_ROOT': '/srv/app/tmp/reva/root', `
before

Reply:
following the same pattern. Or was it wrong before?